setup.sh

```diff
@@ -2,8 +2,7 @@
 
 # check ptrace_scope for PIN
 if ! grep -qF "0" /proc/sys/kernel/yama/ptrace_scope; then
-  echo "Please run 'echo 0|sudo tee /proc/sys/kernel/yama/ptrace_scope'"
```

Comment:
Could you revert it?
We don't want to change kernel configuration without warning.

Reply:
yep, I didn't mean to leave this in, but it was necessary to enable the CI.